doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -306,6 +306,7 @@ If `data_model=dm-by-entity` and `attr_persistence=column` then `NGSIMongoSink`
 | mongo\_username | no | <i>empty</i> | If empty, no authentication is done. |
 | mongo\_password | no | <i>empty</i> | If empty, no authentication is done. |
 | mongo\_auth_source | no | <i>empty</i> | Auth source database use to authenticate the user. Usually could be `admin`. |
+| mongo\_replica_set | no | <i>empty</i> | Replica Set name. Usually could be `replica`. |
```

Comment:
I think is would be a good idea to remark the optionality of this parameter, the following way:

```suggestion
| mongo\_replica_set | no | <i>empty</i> | Replica Set name. Note that this parameter is optional because Cygnus is able to connect to a MongoDB replica set without needing to specify its name. |
```

With regards to `replica` I think is not so usual :) so better not mention about it.

Reply:
Fixed in cba084e